src/streaming/text/TextSourceBuffer.js

```diff
@@ -348,6 +348,8 @@ function TextSourceBuffer() {
                             result = parser.parse(ccContent, offsetTime, sampleStart / timescale, (sampleStart + sample.duration) / timescale, images);
                             textTracks.addCaptions(currFragmentedTrackIdx, firstSubtitleStart / timescale, result);
                         } catch (e) {
+                            fragmentModel.removeExecutedRequestsBeforeTime(Date.now());
```

Comment:
No need to call Date.now(), an empty argument will remove all the executed requests.

Reply:
That would cause a comparison of a `number` and `undefined - number`, that's `number >= NaN`. I don't really like that, but I'll change it if it's necessary

Reply:
Hi @aescarcha ,

sorry I don't see where the problem is. If time is undefined, the filter function of the array will return false. Do I miss something?

Nico

Reply:
Hi @nicosang It's just that it looks a bit ugly for me. I'll remove it now.